packages/dds/sequence/src/defaultMap.ts

```diff
@@ -399,7 +393,7 @@ export class DefaultMap<T> {
 				const localValue = this.data.get(op.key) ?? this.createCore(op.key, local);
 				const handler = localValue.getOpHandler(op.value.opName);
 				const previousValue = localValue.value;
-				const translatedValue = parseHandles(op.value.value, this.serializer);
+				const translatedValue = op.value.value as any;
```

Comment:
All this ValueType stuff is new to me - can you just tell me where this `parseHandles` moves to in the PR? I traced through to see that the `message` param has already been parsed, but I don't know where `op` comes from (and trying to move through the PR)

Reply:
Ahh I spoke too soon :) The `emit` function defined below in `makeMapValueOpEmitter` no longer encodes the handles.  I'm assuming that this inner nugget will have its handles encoded when the whole op goes to `SharedObject.submitLocalMessage`?  And this code path has test coverage in the new functional test cases you're adding?

Reply:
> this code path has test coverage

I could be wrong but I put a breakpoint in the `emit` function below and ran the tests and it didn't get hit.  Please double-check that this has test coverage (though for the record I do agree the change looks right)